README.md

```diff
@@ -90,17 +97,341 @@ for page transitions, and scrollable content within each page.
+  }
+
+  return MaterialApp(
+    themeMode: _isLightTheme ? ThemeMode.light : ThemeMode.dark,
+    theme: ThemeData.light(useMaterial3: true).copyWith(
+      extensions: const <ThemeExtension>[
+        WoltModalSheetThemeData(
+          heroImageHeight: _heroImageHeight,
+          topBarShadowColor: _lightThemeShadowColor,
+          modalBarrierColor: Colors.black54,
+          mainContentScrollPhysics: ClampingScrollPhysics(),
+        ),
+      ],
+    ),
+    darkTheme: ThemeData.dark(useMaterial3: true).copyWith(
+      extensions: const <ThemeExtension>[
+        WoltModalSheetThemeData(
+          topBarShadowColor: _darkThemeShadowColor,
+          modalBarrierColor: Colors.white12,
+          sabGradientColor: _darkSabGradientColor,
+          mainContentScrollPhysics: ClampingScrollPhysics(),
+        ),
+      ],
+    ),
+    home: Scaffold(
+      body: Builder(
+        builder: (context) {
+          return Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Row(...),
+              ElevatedButton(
+                onPressed: () {
+                  WoltModalSheet.show<void>(
+                    context: context,
+                    pageListBuilder: (modalSheetContext) {
+                      final textTheme = Theme.of(context).textTheme;
+                      return [
+                        page1(modalSheetContext, textTheme),
+                        page2(modalSheetContext, textTheme),
+                      ];
+                    },
+                    modalTypeBuilder: (context) {
+                      final size = MediaQuery.sizeOf(context).width;
+                      if (size < _pageBreakpoint) {
+                        return WoltModalType.bottomSheet();
+                      } else {
+                        return WoltModalType.dialog();
+                      }
+                    },
+                    onModalDismissedWithBarrierTap: () {
+                      debugPrint('Closed modal sheet with barrier tap');
+                      Navigator.of(context).pop();
+                    },
+                  );
+                },
+                child: const SizedBox(
+                  height: _buttonHeight,
+                  width: _buttonWidth,
+                  child: Center(child: Text('Show Modal Sheet')),
+                ),
+              ),
+            ],
+          );
+        },
+      ),
+    ),
+  );
+}
+```
+
+The example project above will create the following:
+</br>
+</br>
+
+![Example app](https://github.com/woltapp/wolt_modal_sheet/blob/main/doc/example_wms_demo.gif?raw=true)
+
+### Playground app with imperative navigation
+
+The [playground](./playground/) app demonstrates how to imperatively show the
+modal sheet. The purpose of this module is to play and experiment with various
+use cases.
+
+### Playground app with declarative navigation
+
+The [playground_navigator2](./playground_navigator2/) has the same content with
```

Comment:
Suggestion:

The [playground_navigator2](./playground_navigator2/) is a variant of the playground application with fewer use cases. In this example, a modal sheet is displayed using the Navigator 2.0 (Router API) in a declarative navigation pattern. As users navigate through the pages, the browser's address bar updates to reflect the name and index number of the current page. Additionally, users can directly modify the visible page by changing the query parameters of the URL in the browser's address bar.